version.go

```diff
@@ -0,0 +1,80 @@
+/*
```

Comment:
Versioning logic is already available in https://github.com/ocibuilder/ocibuilder/blob/master/provenance/provenance.go

Reply:
We should append anything that you see missing there

Reply:
Cool, didn't know it was there